### ISSUE TITLE
Best posible training / validation and test datasplit set.

### ISSUE THREAD

**Issue by Goru1890** [AlexeyAB/darknet#7048]
As written in this [article](https://medium.com/@oviyum/lessons-from-fine-tuning-a-convolutional-binary-classifier-ccf9388e46d8) and having a conclusion that the data split is crucial, I am curious to know which is the theoric best possible dataset split for YOLO.

Which is your experience?

**Comment by readicculus** [comment]
Lots of ways.  If have less data then your test set might be smaller than ideal.  General rule of thumb is around 70-80% train rest test if only test and train set.
If you are splitting into test/train valid then usually want around 5-10% valid then 20-30% test then 50-60% or so train.
Really depends on how much data you have as data is often very limited in real world applications.

Also try and have a similar distribution of classes or object densities between test/train and the whole dataset.
If you have 1,000 cats and 5,000 dogs then the training, test, and/or validation sets should aim to have about a ~1:5 ratio of cats to dogs if that makes sense.
